packages/node_modules/@webex/internal-plugin-devices/test/unit/spec/features/feature-model.js

```diff
@@ -56,6 +56,113 @@ describe('plugin-devices', () => {
       });
     });
 
+    describe('#parse()', () => {
+      let fixture;
+      let model;
+
+      beforeEach('initialize the feature model', () => {
+        featureModel = new FeatureModel();
+      });
+
+      it('should return an empty object when the model is not defined', () => {
+        model = featureModel.parse(fixture);
+        assert.deepEqual(model, {});
+      });
+
+      describe('when the model is defined', () => {
+        beforeEach('define the fixture', () => {
+          fixture = {};
+        });
+
+        describe('when the value is a number', () => {
+          let num;
+
+          beforeEach('set the value to a number', () => {
+            fixture.val = '1234';
+            num = Number(fixture.val);
+            model = featureModel.parse(fixture);
+          });
+
+          it('should set the value to a instance of number', () =>
+            assert.typeOf(model.value, 'number'));
+
+          it('should set the type to \'number\'', () =>
+            assert.equal(model.type, 'number'));
+
+          it('should set the value to match the pto', () =>
```

Comment:
This test description needs to be a bit more specific. Something like "should set the model value to the equivalent Number value".

Reply:
Changed this statement to match your provided statement.